tmol/tests/support/test_database_converters.py

```diff
@@ -32,7 +33,7 @@ def test_rama_table_read(rosetta_database, default_database):
     )
     r3_paa_dir = os.path.join(rosetta_database, "scoring/score_functions/P_AA_pp/")
 
-    r3_general, r3_prepro = parse_all_tables(
+    r3_general, r3_prepro = rewrite_rama_binary.parse_all_tables(
```

Comment:
What would it take to get these tests running on the testing server?

Reply:
So all it needs is pyrosetta installed as a dependency.  Not sure if that is worth it?